tools/ola_trigger/config.lex

```diff
@@ -40,7 +40,7 @@ whitespace   [ \t]
 ^#[^\n]*             {}
 {whitespace}+#[^\n]* {}
 {whitespace}+     { count(); return WHITESPACE; }
-\n                { count(); yylineno++; return NEW_LINE; }
+\n                { count(); return NEW_LINE; }
```

Comment:
I found with this in place, the line numbers were wrong; with it removed, they were correct. It's probably worth testing on some other OSes though.

I think we could remove our yywrap function too, and just use the noyywrap option if it's present in old enough releases.